### PR TITLE
Add "Package:" tag with packages used on the test

### PR DESCRIPTION
**Pull request by emiura**
Package tag will be later parsed and added to a page like http://qam.suse.de/openqa-summary-tests/Maintenance_SLE_15_SP1_Updates.html.

Related ticket poo#77002

### CODE REVIEW

File: tests/console/bind.pm [hunk 1]
```diff
@@ -7,6 +7,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: libopenssl-devel libmysqlclient-devel bind rpm-build perl-IO-Socket-INET6
+# bind rpm-build bind-utils net-tools-deprecated perl-IO-Socket-INET6 perl-Socket6 perl-Net-DNS python3-dnspython
```
Comment (ggkioulis) [inline tests/console/bind.pm:11]: Are we sure that [openqa-summary-tests](https://gitlab.suse.de/qa-maintenance/openqa-summary-tests) parses the second line as well?

File: tests/console/shells.pm [hunk 2]
```diff
@@ -32,7 +33,7 @@ sub run() {
     select_console 'root-console';
     my @packages = qw(tcsh zsh);
     # ksh does not build for Leap 15.x on aarch64, so, skip it
-    push @packages, qw(ksh) unless (is_leap('15.0+') and check_var('ARCH', 'aarch64'));
+    tcsh zshpush @packages, qw(ksh) unless (is_leap('15.0+') and check_var('ARCH', 'aarch64'));
```
Comment (ggkioulis) [inline tests/console/shells.pm:36]: Maybe this could be in a separate PR.

File: tests/console/zypper_ref.pm [hunk 3]
```diff
@@ -8,6 +8,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: Zypper
```
Comment (ggkioulis) [inline tests/console/zypper_ref.pm:11]:
```suggestion
# Package: zypper
```

File: tests/console/apache.pm [hunk 4]
```diff
@@ -7,6 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package apache2 apache2-mod_php72 php72-php72-curl
```
Comment (ggkioulis) [inline tests/console/apache.pm:10]:
```suggestion
# Package: apache2 apache2-mod_php72 php72-php72-curl
```

File: tests/console/bind.pm (hunk 1 above)
Comment (emiura) [reply tests/console/bind.pm:11]: According to Marcelo, it parses the text until it finds the next tag.

File: tests/console/shells.pm (hunk 2 above)
Comment (emiura) [reply tests/console/shells.pm:36]: This looks like an "oops", I don't know where this came from.

File: tests/console/avocado_run.pm [hunk 5]
```diff
@@ -7,6 +7,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: avocado
```
Comment (dzedro) [inline tests/console/avocado_run.pm:10]: Avocado is used to test qemu.

File: tests/console/install_packages.pm [hunk 6]
```diff
@@ -8,6 +8,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: perl-solv perl-Data-Dump
```
Comment (dzedro) [inline tests/console/install_packages.pm:11]: zypper libzypp and don't know what else is involved

File: tests/console/ntp_client.pm [hunk 7]
```diff
@@ -5,6 +5,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: chrony
```
Comment (dzedro) [inline tests/console/ntp_client.pm:8]: & ntp

**Review by ggkioulis — APPROVED** [review]
LGTM! Thank you for this!
